### ISSUE TITLE
Accessibility issues, item images have no alt text - Mac voiceover

### ISSUE THREAD

**Issue by deltanetdan** [deltanet/adapt-blinds#19]
The component does have a field for alt text, but it has not been used. Screen reader user just gets button for each option. 

![blinds-Picture 1](https://user-images.githubusercontent.com/4161219/120505571-fe246c80-c3bc-11eb-8411-7bb82379baef.png)

If you add an aria label to the button the text becomes inaccessible through the Keyboard and is not read by screen reader. 

After activating a button, the focus should shift to the item text. Also, the item text is not navigable.

![blinds-Picture2](https://user-images.githubusercontent.com/4161219/120505630-10060f80-c3bd-11eb-8abc-15bd804a73a1.png)

**Comment by deltanetdan** [comment]
Changes in [a7a36e7](https://github.com/deltanet/adapt-blinds/commit/a7a36e7387e1e83806167b06a0f005ef2373c76f) don't move focus to the text in WIN 10 IE11 and Firefox (works in chrome and edge on mac). Probably because the text is in the button element and the screen reader considers it a button not a text element.

item title and text should probably be moved out of the button tag and positioned using CSS.
